rmgpy/reaction.py

```diff
@@ -339,6 +343,14 @@ def to_cantera(self, species_list=None, use_chemkin_identifier=False):
                         reactants=ct_reactants, products=ct_products, rate=rate
                     )
 
+            elif isinstance(self.kinetics, SurfaceArrhenius):
+                rate = self.kinetics.to_cantera_kinetics()
+                ct_reaction = ct.InterfaceReaction(equation=str(self), rate=rate)
+
+            elif isinstance(self.kinetics, StickingCoefficient):
+                rate = self.kinetics.to_cantera_kinetics()
+                ct_reaction = ct.Reaction(equation=str(self), rate=rate)
+
```

Comment:
This declaration should probably match the way all the other Cantera reactions are declared in this section.
```suggestion
            elif isinstance(self.kinetics, StickingCoefficient):
                ct_reaction = ct.Reaction(reactants=ct_reactants, products=ct_products, rate=ct.StickingArrheniusRate())

```

Reply:
Thanks. resolved in a subsequent commit